src/shared-components/tooltip/index.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import OffClickWrapper from '../offClickWrapper';
+import { MainContainer, Trigger, TooltipBox } from './style';
+
+const propTypes = {
+  alignRightPercent: PropTypes.number,
+  alignTopPercent: PropTypes.number,
+  arrowAlign: PropTypes.oneOf(['left', 'middle', 'right']),
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]).isRequired,
+  content: PropTypes.node,
+  nudgeLeft: PropTypes.number,
+  nudgeRight: PropTypes.number,
+  nudgeTop: PropTypes.number,
+  nudgeBottom: PropTypes.number,
+  position: PropTypes.oneOf(['top', 'bottom']),
+  defaultOpen: PropTypes.bool,
+  display: PropTypes.bool,
+};
+
+const defaultProps = {
+  arrowAlign: 'middle',
+  content: '',
+  nudgeLeft: 0,
+  nudgeRight: 0,
+  nudgeTop: 0,
+  nudgeBottom: 0,
+  position: 'top',
+  defaultOpen: false,
+  display: true,
+};
+
+class Tooltip extends React.Component {
+  state = { clicked: false, hovered: false };
+
+  onMouseEnter = () => {
+    this.setState({ hovered: true });
+  };
+
+  onMouseLeave = () => {
+    this.setState({ hovered: false });
+  };
+
+  onClick = () => {
+    const { clicked } = this.state;
+
+    if (clicked) {
+      // if clicked is true, we're about to update to false so remove class
+      document.querySelector('body').classList.remove('cursor-pointer');
```

Comment:
Since `cursor-pointer` css class isn't a thing, you'll want to inject it like we do here https://github.com/PocketDerm/PocketDerm/blob/28ad86f826ce599fd9171332c5c768218c274547/resources/assets/jsx/dashboard/components/shipmentRescheduler/index.js#L182-L188

Reply:
right,done. Thanks for catching this up, I understand now, tested it works.